src/main.cpp

```diff
@@ -25,21 +26,21 @@ void SonicGenCleanup();
 
 int main ( int argc, char *argv[] )
 {
-    cout << "Sonic/C++ translator - Copyright (C) 1998 by Don Cross <cosinekitty@gmail.com>\n";
-    cout << "Version " << SONIC_VERSION << ", released on " << SONIC_RELEASE_DATE << ".\n";
-    cout << "https://github.com/cosinekitty/sonic" << "\n\n";
+    std::cout << "Sonic/C++ translator - Copyright (C) 1998 by Don Cross <cosinekitty@gmail.com>\n";

```

Comment:
I think it would be OK inside the scope of a function to put a `using namespace std;`.  I agree with you about not doing it at global scope.